src/js/pensions/config/form.js

```diff
@@ -614,6 +614,7 @@ const formConfig = {
                 showFieldLabel: true,
                 keepInPageOnReview: true,
                 expandUnder: 'maritalStatus',
+                useLabel: true,
```

Comment:
Because `showFieldLabel` is `true`? (Also, why is that part of the determination to use a `<legend>` or not?)

Reply:
Yeah, I suppose that's a little convoluted. Most of the time when `showFieldLabel` is true, it's because we want to show a label for an object property, which is going to be a label for a group of elements, and therefore should be a `legend`. `useLabel` is an option to force the template to use a `label` instead of a `legend`.

So `showFieldLabel` is using the word "label" generically, and `useLabel` is using it specifically to refer to `label` elements. Which is confusing, yeah.

`useLabel` could be `useLabelInsteadOfLegend`? I'm open to suggestions.

Reply:
Is it possible to consolidate this logic into one option, such as labelType, and specify label legend or none?

Reply:
A `labelType` option would work, but I'm not sure I want to force us to choose each time we use this option. Normally you would want to let the FieldTemplate component choose the right one.

I'm going to change `showFieldLabel` to allow a 'label' string for when you want a label element, I think that's actually clearer.